<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transmitter</class>
 <widget class="QWidget" name="transmitter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1616</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="labelSelector">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="equipmentSelector">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Equipment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Label:</string>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="addLabel">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeLabel">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="lArincBitRate">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chanel Bit Rate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cArinc_parity_bitRate">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pLoadConfig">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pSaveConfig">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>600</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSelectedFile">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>610</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No file selected</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chTransmitterEnabled">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>610</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chTransmitterDisabled">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>610</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
